stmhal/boards/PYBV3/mpconfigboard.h

```diff
@@ -15,6 +16,7 @@
 #define MICROPY_HW_ENABLE_DAC       (1)
 #define MICROPY_HW_ENABLE_I2C1      (1)
 #define MICROPY_HW_ENABLE_SPI1      (1)
+#define MICROPY_HW_HAS_LED          (1)
```

Comment:
Both variants of the SPI2_MISO pins are used for hard-coded functions (the onboard line detectors), so SPI2 can't be used. - Oh yeah - copy/paste error - I see it.